tests/cupy_tests/manipulation_tests/test_kind.py

```diff
@@ -10,6 +10,21 @@
 @testing.gpu
 class TestKind(unittest.TestCase):
 
+    @testing.for_orders('CFAK')
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal()
+    def test_asarray_chkfinite(self, xp, dtype, order):
+        a = [0, 4, 0, 5]
+        return xp.asarray_chkfinite(a, dtype=dtype, order=order)
+
+    @testing.for_orders('CFAK')
+    @testing.for_all_dtypes()
+    def test_asarray_chkfinite_non_finite_vals(self, xp, dtype, order):
```

Comment:
We don't use the decorator `numpy_cupy_array_equal()` here, so `xp` is not needed:
```suggestion
    def test_asarray_chkfinite_non_finite_vals(self, dtype, order):
```